plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestSqlServerConnectorTest.java

```diff
@@ -171,6 +191,11 @@ private int getTableOperationsCount(String operation, String table)
         }
     }
 
+    private static String doubleQuote(String name)
+    {
+        return "\"" + name + "\"";
+    }
```

Comment:
For now I'd recommend inlining this since there are lot of other tests which do same thing.

As a follow-up you can add a static helper + replace other usages in one go.